infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/OpenShiftEnvironmentProvisioner.java

```diff
@@ -85,9 +88,10 @@ public OpenShiftEnvironmentProvisioner(
       WorkspaceVolumesStrategy volumesStrategy,
       ContainerResourceProvisioner resourceLimitRequestProvisioner,
       LogsVolumeMachineProvisioner logsVolumeMachineProvisioner,
-      PodTerminationGracePeriodProvisioner podTerminationGracePeriodProvisioner,
+      OpenShiftPodTerminationGracePeriodProvisioner podTerminationGracePeriodProvisioner,
```

Comment:
could you clarify why do we need `OpenShiftPodTerminationGracePeriodProvisioner` and can not handle the logic in the original `PodTerminationGracePeriodProvisioner` ?

Reply:
Pure Kubernetes environment not target platform for Asynchronous storage feature

Reply:
hmm.. why?

Reply:
After internal discussion decided to remove `OpenShiftPodTerminationGracePeriodProvisioner`and provide changes in original `PodTerminationGracePeriodProvisioner`